5.6/windows/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# escape=`
```

Comment:
What is this for?

Reply:
Ah, it is needed to get the line continuation to work.  I don't fully understand the _why_, but this is actually a control message - similar to a shebang.